created views

MyskYko opened the pull request:
タイムラインと投稿画面のビューを作って遷移できるところまで作りました。
![timeline](https://cloud.githubusercontent.com/assets/18373300/25337219/f8df1ac0-2935-11e7-939c-be56ae1315b6.png)
![tweetmaking](https://cloud.githubusercontent.com/assets/18373300/25337222/faa5fd06-2935-11e7-83aa-f06f29fa6cde.png)

MyskYko committed:
created views

yukitoto commented on Miyatter/MakeTweetViewController.swift:
```diff
@@ -0,0 +1,101 @@
+        button.backgroundColor = UIColor.lightGray
+        return button
+    }()
+    
```
MARKの上には２行あけるようにしよう

yukitoto commented on Miyatter/MakeTweetViewController.swift:
```diff
@@ -0,0 +1,101 @@
+        let text = UITextView()
+        text.layer.borderColor = UIColor.darkGray.cgColor
+        text.layer.borderWidth = 1
+        text.textAlignment = NSTextAlignment.left
```
`text.textAlignment = .left`

こうしよう

yukitoto commented on Miyatter/MakeTweetViewController.swift:
```diff
@@ -0,0 +1,101 @@
+        let button = UIButton()
+        button.setTitle("戻る", for: .normal)
+        button.titleLabel?.font = UIFont(name: "HiraKakuProN-W3", size: 20)
+        button.addTarget(self, action: #selector(tapBackButton), for: .touchUpInside)
```
addTargetじゃなくて、`rx.tap`を使うようにしよう

yukitoto commented on Miyatter/MakeTweetViewController.swift:
```diff
@@ -0,0 +1,101 @@
+    // MARK: - View Transition -
+    
+    func tapBackButton() {
+        self.dismiss(animated: true, completion: nil)
```
self無しで

yukitoto commented on Miyatter/MakeTweetViewController.swift:
```diff
@@ -0,0 +1,101 @@
+        }
+    }
+    
+    // MARK: - View Transition -
```
いいね

yukitoto commented on Miyatter/TimeLineViewController.swift:
```diff
@@ -0,0 +1,104 @@
+    
+    private lazy var tableView: UITableView = {
+        let table = UITableView()
+        table.register(TweetCell.self, forCellReuseIdentifier: "tweetCell")
```
クラス名とIdentifierを揃えよう（コード規約）

yukitoto replied:
`table.register(TweetCell.self, forCellReuseIdentifier: "TweetCell")`

yukitoto commented on Miyatter/TimeLineViewController.swift:
```diff
@@ -0,0 +1,104 @@
+    // MARK: - View Transition -
+    
+    func tapTweetButton() {
+        self.present(MakeTweetViewController(), animated: true, completion: nil)
```
selfをなくそう

yukitoto commented on Miyatter/TimeLineViewController.swift:
```diff
@@ -0,0 +1,104 @@
+    
+    // MARK: - View -
+    
+    private lazy var headerView: UIView = {
```
privateじゃなくてfileprivateを使うようにしたい

yukitoto commented on Miyatter/TweetCell.swift:
```diff
@@ -0,0 +1,79 @@
+    
+    // MARK: - View -
+    
+    lazy var dateLabel: UILabel = {
```
fileprivateでいいんじゃないかな？

myskyko replied:
tableViewからの呼び出しでtextプロパティをいじるのでprivateにできません

yukitoto replied:
そういう場合はOK！
ひとまずこのままでいいんだけど、このあと`Post`モデルを作ってそのインスタンスのデータを表示するようになると思うんだけど、そうなったら
```TweetCell.swift

func update(post: Post)
  dateLabel.text = post.createdAt
end
```
みたいな感じにした方がViewControllerに不要なデータ表示用のロジックを書く必要がなくなって良いと思うから変更していこう。この書き方になると、外から参照する必要性がなくなるので、privateでも大丈夫になる。

MyskYko committed:
modified
 - fileprivate
 - use rx
 - textalignment.left to .left
 - identifier is changed equal to classname
 - some indent

MyskYko commented:
@yukitoto 
訂正しました

## やったこと
 - fileprivate
 - rxを使ったボタンアクション
 - .leftへ
 - identifierをクラス名と揃えました
 - indentを少し整えました

yukitoto commented:
こういう形でプルリク送ってもらえると、めっっっっちゃ助かるわ。
あざす

yukitoto reviewed (commented):
rxの使いかた、今ので良いと思う！お疲れ様です！
